src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -1894,7 +1890,7 @@ private static IExternalSerializer GetFallbackSerializer(bool useJsonSerializer)
 #if NETSTANDARD
                 throw new OrleansException("Can't use binary formatter as fallback serializer while running on .Net Core");
 #else
-                serializer = new BinaryFormatterSerializer();
+                serializer = new IlBasedFallbackSerializer(new IlBasedSerializers());
```

Comment:
For now, how about I change the code to the following?
```C#
if (useJsonSerializer)
{
    serializer = new OrleansJsonSerializer();
}
else
{
#if NETSTANDARD
    serializer = new IlBasedFallbackSerializer(new IlBasedSerializers());
#else
    serializer = new BinaryFormatterSerializer();
#endif
}
```
Then, in a following commit, I'll remove useJsonSerializer and have the correct serializer passed in through the `Initialize`/`InitializeForTesting` method, which we'll choose using a config option.

Alternatively, I can do it all in this PR